code/controllers/Processes/power.dm

```diff
@@ -1,14 +1,30 @@
 var/global/list/power_machinery_profiling = list()
 var/global/list/power_machines = list()
+var/global/list/datum/powernet/powernets = list() //Holds all powernet datums in use or pooled
+var/global/list/cable_list = list() //Index for all cables, so that powernets don't have to look through the entire world all the time
 
 /datum/controller/process/power_machinery
-	schedule_interval = 37 // every 2 seconds
+	schedule_interval = 17 // every 2 seconds
```

Comment:
Oh, I didn't really bother to look at the diff, but I could've sworn they were at .6 before.